search/includes/classes/class-queue.php

```diff
@@ -596,6 +599,32 @@ public function get_jobs( $job_ids ) {
 		return $jobs;
 	}
 
+	/**
+	 * Get the jobs using the lowest and highest job_id range
+	 */
+	public function get_jobs_by_range( $min_id, $max_id ) {
+		global $wpdb;
+
+		if ( ! $min_id || ! $max_id ) {
+			return array();
+		}
+
+		$table_name = $this->schema->get_table_name();
```

Comment:
Given that we can't prepare the table name, would it make sense to further sanitize this variable?

Reply:
We could yeah. But then again this is not a user-input and I think it is a pretty standard pattern we use all over (without sanitization). So I would lean towards not adding it. But I am happy to be convinced otherwise:)

Reply:
It could be content that comes from a database or a third-party service (or any dynamic source, for that matter). In that case we should treat it as user input. 

In this case, it comes from a hardcoded string in a class, so I guess we're good.